src/applications/mhv/medical-records/components/shared/DownloadingRecordsInfo.jsx

```diff
@@ -20,18 +20,7 @@ const DownloadingRecordsInfo = props => {
     >
       {/* Use the "!== false" syntax because checking the inverse causes the component to be resized incorrectly
             when the value changes from undefined to true. */}
-      {allowTxtDownloads !== false ? (
-        <ul>
-          <li>{publicSharedContent()}</li>
-          <li>
-            <strong>If you use assistive technology,</strong> a Text file (.txt)
-            may work better for technology such as screen reader, screen
-            enlargers, or Braille displays.
-          </li>
-        </ul>
-      ) : (
-        <p>{publicSharedContent()}</p>
-      )}
+      {allowTxtDownloads !== false ? <p>{publicSharedContent()}</p> : null}
```

Comment:
It looks like you may have inadvertently changed the logic of this block.

Reply:
I have return the code to the previous code base to correct the logic error.